swagger.yaml

```diff
@@ -0,0 +1,129 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Tech Holding Classes
+  description: An app to manage Tutorials
+host: localhost:8000
+basePath: "/"
+tags:
+- name: tutorials
+  description: API for user in the system
+schemes:
+- http
+consumes:
+- application/json
+produces:
+- application/json
+paths:
+  tutorial:
+    parameters:
+    - name: id
+      in: path
+      required: true
+      description: ID of tutorial that we want to find
+      type: integer
```

Comment:
The id in mongodb is `UUID` which is a type of string. Please use type as string and format as `uuid`, refer to this document: https://swagger.io/docs/specification/data-models/data-types/

CC: @JainishaParmar @ashishkava